package/kedro_viz/server.py

```diff
@@ -320,15 +321,16 @@ def _call_viz(
     global data  # pylint: disable=global-statement,invalid-name
 
     if load_file:
-        import logging
         # Remove all handlers for root logger
         root_logger = logging.getLogger()
         root_logger.handlers = []
 
         data = _load_from_file(load_file)
     else:
         if match(kedro.__version__, ">=0.15.0"):
-            from kedro.context import KedroContextError
+            from kedro.context import (  # pylint: disable=import-outside-toplevel
```

Comment:
Super minor, but maybe move the disable comment to above the import line, rather than on the same line and breaking it?